tests/Parse/ParseSchemaTest.php

```diff
@@ -449,8 +466,108 @@ public function testBadSchemaDelete()
         $user->setUsername('schema-user');
         $user->setPassword('basicpassword');
         $user->signUp();
-
         $schema = new ParseSchema(self::$badClassName);
         $schema->delete();
     }
+
+    public function testCreateIndexSchemaStream()
```

Comment:
I would consolidate these pair tests into singular tests. You don't need to manually test each as part of the CI selectively runs the entire test suite against the stream client in addition to the curl client. The other reason being if we ever add additional clients in the future we would have to add additional tests if we did it this way, versus just adding another CI run selectively using the alternate client instead. 

There are _some_ tests in here that still do this and it was probably me who added them. Those were early tests before I had setup the independent curl/stream CI runs. Honestly they can be selectively phased out at now. But to sum it up we shouldn't add additional stream differing tests (unless there's an extenuating circumstance) as it's redundant now.

Reply:
Just to detail where it is you can see the stream client testing in the matrix for [**5.4** and **HHVM**](https://github.com/parse-community/parse-php-sdk/blob/master/.travis.yml#L17...L22) and later you can see how it is actually factored into the tests [here](https://github.com/parse-community/parse-php-sdk/blob/master/.travis.yml#L40...L41).